tests/app/shared/issue/label/label.component.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { LabelComponent } from '../../../../../src/app/shared/issue/label/label.component';
+import { LabelService } from '../../../../../src/app/core/services/label.service';
+import { PermissionService } from '../../../../../src/app/core/services/permission.service';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../../constants/githubissue.constants';
+import { Issue } from '../../../../../src/app/core/models/issue.model';
+import { SEVERITY_LABELS, COLOR_SEVERITY_LOW, SEVERITY, COLOR_SEVERITY_HIGH, SEVERITY_HIGH } from '../../../../constants/label.constants';
+import { of } from 'rxjs';
+
+describe('LabelComponent', () => {
+  let labelComponent: any;
+  let issueService: any;
+  let permissionService: any;
+  let labelService: any;
+  let thisIssue: Issue;
+  let issueUpdatedEmit: any;
+
+  const formBuilder: any = null;
```

Comment:
Same here, let's do away with these formBuilder and errorHandlingService variables

Reply:
Alright 👍